deps/unwind.mk

```diff
@@ -4,8 +4,15 @@ ifneq ($(USE_BINARYBUILDER_LIBUNWIND),1)
 LIBUNWIND_CFLAGS := -U_FORTIFY_SOURCE $(fPIC)
 LIBUNWIND_CPPFLAGS :=
 
+# XXX: Kludge for libunwind 1.5.0 tag name not being the full version
```

Comment:
Could we open an issue there and ask them to push `v1.5.0` as a tag to github?

Reply:
https://github.com/libunwind/libunwind/issues/309

Reply:
Fixed